src/overlays/actors/ovl_En_Po_Sisters/z_en_po_sisters.c

```diff
@@ -97,46 +97,46 @@ static ColliderCylinderInit sCylinderInit = {
 static CollisionCheckInfoInit sColChkInfoInit = { 6, 25, 60, 50 };
 
 typedef enum {
-    /* 0x0 */ POSISTERS_DAMAGEEFFECT_NOEFFECT,
-    /* 0x1 */ POSISTERS_DAMAGEEFFECT_UNKDMG12, // set in DamageTable, but unused
-    /* 0x4 */ POSISTERS_DAMAGEEFFECT_LIGHTARROWS = 0x4,
-    /* 0xE */ POSISTERS_DAMAGEEFFECT_SPINATTACK = 0xE,
-    /* 0xF */ POSISTERS_DAMAGEEFFECT_DEKUNUT = 0xF,
+    /* 0x0 */ POSISTERS_DMGEFF_NONE,
+    /* 0x1 */ POSISTERS_DMGEFF_UNKDMG12, // set in DamageTable, but unused
+    /* 0x4 */ POSISTERS_DMGEFF_LIGHTARROWS = 0x4,
+    /* 0xE */ POSISTERS_DMGEFF_SPINATTACK = 0xE,
+    /* 0xF */ POSISTERS_DMGEFF_DEKUNUT = 0xF,
 } PoSisterDamageEffect;
```

Comment:
```suggestion
    /* 0x0 */ POE_SISTERS_DMGEFF_NONE,
    /* 0x1 */ POE_SISTERS_DMGEFF_UNKDMG12, // set in DamageTable, but unused
    /* 0x4 */ POE_SISTERS_DMGEFF_LIGHTARROWS = 0x4,
    /* 0xE */ POE_SISTERS_DMGEFF_SPINATTACK = 0xE,
    /* 0xF */ POE_SISTERS_DMGEFF_DEKUNUT = 0xF,
} PoeSisterDamageEffect;
```
If you want, although it's not the actor of focus so no worries if you want to leave it